gst/mqtt/meson.build

```diff
@@ -11,4 +11,6 @@ gstmqtt_shared = shared_library('gstmqtt',
   install_dir: plugins_install_dir
 )
 
-gstmqtt_dep = declare_dependency(link_with: gstmqtt_shared)
+gstmqtt_dep = declare_dependency(
```

Comment:
I understood why you've removed gstmqtt_shared, but added include_directories only.
However, a few months later, someone might think this is weird and try to "fix" it. Please add a comment here why gstmqtt_dep looks like this. (its users will mock.)

And.. if you want better (following recent trend), it'd be better if you rename ```gstmqtt_dep``` as ```gstmqtt_mocking_dep```.

Reply:
Brief comments have been added.